aws/metaData.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+func GetMetaData(path string) (contents []byte, err error) {
+	url := "http://169.254.169.254/latest/meta-data/" + path
```

Comment:
It also is limiting in that only the "latest" is returned, no versioning is allowed.

Perhaps the url prefix should just be http://169.254.169.254/ ?  I am fine w/ just http://169.254.169.254/latest/ .

We need to walk that line between usability and flexibility while not bloating the package with a ton of methods.